relay-server/src/envelope/item.rs

```diff
@@ -191,6 +192,35 @@ impl Item {
         }
     }
 
+    /// Returns the number of spans in `event.spans`.
+    pub fn span_count(&self) -> usize {
+        self.headers.span_count.unwrap_or(0)
+    }
+
+    #[cfg(test)]
+    pub fn set_span_count(&mut self, value: usize) {
```

Comment:
Generally nice to have some docs, even if it's just available for tests. Calling out it is just for tests and why it isn't available otherwise (e.g. should use `refresh...` instead).